test/circuitbreaker.test.js

```diff
@@ -371,4 +371,53 @@ describe('Circuit Breaker', function(){
     });
 
   });
+
+  describe('errorFn', function () {
+    it('should break based on error function parameter', function (done) {
+      var breakerFn = function(id, callback) {
```

Comment:
Feel free to break backwards compatibility with node 0.8 and node 0.10. Use ES2015.